pypeit/scripts/collate_1d.py

```diff
@@ -505,6 +514,11 @@ def get_parser(cls, width=None):
                                  'F|                        "ra/dec"\n'
                                  'F|  dry_run               If set the matches are displayed\n'
                                  'F|                        without any processing\n'
+                                 'F|  flux                  Flux calibrate using archived sensfuncs.\n'
+                                 'F|  ignore_flux           Ignore any flux calibration information in\n'
+                                 'F|                        spec1d files.\n'
+                                 'F|  rms_thresh            If set any objects with rms > than this are\n'
+                                 'F|                        skipped.\n'
```

Comment:
I would suggest to rephrase as "If set, any objects with a rms > than the input value are skipped, else all are accepted."

Reply:
I went with " If set, any objects with a rms > than the input value are skipped, else all rms values are accepted." in order to clarify that objects might still be rejected for other reasons.